src/main/kotlin/no/nav/hjelpemidler/joark/service/JoarkDataSink.kt

```diff
@@ -44,6 +43,12 @@ internal class JoarkDataSink(
 
     init {
         River(rapidsConnection).apply {
+            validate { it.demandValue("eventName", "hm-Søknad") }
+            validate { it.requireKey("fodselNrBruker", "navnBruker", "soknad", "soknadId") }
+        }.register(this)
+
+        River(rapidsConnection).apply {
```

Comment:
La til en ekstra river som bruker samme target-klasse, slik at vi nå har én validator for hvert innkommende eventNavn. Tror det skal funke og er relativt greit. Virket litt dårlig støttet med å ha flere demandValue's der bare én av de må være satt. Det eksisterer en demandAllOrAny, men den ser ut til å være for json-keys med en array av values. Så da prøver jeg dette tenker jeg.

@olesls

Reply:
👍

Reply:
Der finst ein `requireAny(key: String, values: List<String>)` som ser ut til å sjekke sånn som vi vil. Kanskje vi kan sende inn ein PR der vi legger til ein tilsvarande `demandAny`

Reply:
God ide! Men gjør det seperat så jeg får dette ut nå. Men legger det på lista mi som en plukkoppgave!